test/unit/org/apache/cassandra/streaming/EntireSSTableStreamingCorrectFilesCountTest.java

```diff
@@ -127,7 +127,7 @@ public void test() throws Exception
 
         int totalNumberOfFiles = session.transfers.get(store.metadata.id).getTotalNumberOfFiles();
 
-        assertEquals(CassandraOutgoingFile.getComponentManifest(sstable).components().size(), totalNumberOfFiles);
+        assertEquals(CassandraOutgoingFile.getComponentManifest(sstable, false).components().size(), totalNumberOfFiles);
```

Comment:
@jasonstack Would it be useful to have a simple test that ensures hard links get cleaned up when streaming completes? (perhaps an additional assertion in one of the other tests...)

Reply:
+1